tests/test_create_model.py

```diff
@@ -260,6 +271,7 @@ def _some_func(self):
         assert a._some_func == 2
 
 
+@pytest.mark.xfail(reason='working on V2', strict=False)
```

Comment:
Lines like this should be removed - e.g. if the test is now passing, we should have no `xfail` decorator.

Reply:
@samuelcolvin thanks for catching this. I've removed the xfail marking from test_create_model_with_slots and pushed to the PR.  Hopefully that's the last issue and this is now ready for merge.